barcalendar.py

```diff
@@ -253,17 +302,19 @@ def write(self):
 END_YEAR = 2027
 LTS_ONLY = True
 
+versions = get_defaults_from_tutor()
+
 # The current versions of everything.  Use the same strings as the keys in the various sections below.
 CURRENT = {
-    "Open edX": "Palm",
+    "Open edX": f"{parse_version_name(versions['OPENEDX_COMMON_VERSION'])}",
```

Comment:
There's no need to use an f-string for a single expression.  You can simplify it:
```suggestion
    "Open edX": parse_version_name(versions['OPENEDX_COMMON_VERSION']),
```